ext/gtest/CMakeLists.txt

```diff
@@ -3,15 +3,16 @@ if(IVW_UNITTESTS OR IVW_INTEGRATION_TESTS)
     # settings on Windows
     set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
-    option(BUILD_GMOCK "Builds the googlemock subproject" OFF)
```

Comment:
This indirectly causes cmake after checking out this branch, since it is already set to off. Easily fixed by turning it on. But it might not be obvious, we should either (or both) output an explanation in cmake (tell the user to turn it on if it is off) or mention it in the changelog

```
CMake Error at cmake/globalmacros.cmake:474 (set_target_properties):
  set_target_properties Can not find target to add properties to: gmock
Call Stack (most recent call first):
  ext/gtest/CMakeLists.txt:11 (ivw_folder)


CMake Error at cmake/globalmacros.cmake:474 (set_target_properties):
  set_target_properties Can not find target to add properties to: gmock_main
Call Stack (most recent call first):
  ext/gtest/CMakeLists.txt:12 (ivw_folder)
```

Reply:
Only for old builds right?

Reply:
Yes